app/src/main/java/com/codesoom/assignment/exception/TaskException.java

```diff
@@ -0,0 +1,12 @@
+package com.codesoom.assignment.exception;
+
+import com.codesoom.assignment.type.ErrorCode;
+import lombok.*;
+
+@Getter
+@Setter
+public class TaskException extends Throwable {
```

Comment:
Throwable은 범위가 너무 넓습니다. `java.lang.Error`도 Throwable에 포함되죠. 이번 기회에 자바의 Throwable 상속 구조에 대해서도 조사해 보세요.

```suggestion
public class TaskException extends RuntimeException {
```

Reply:
블로그에 남겼습니다. https://kckoh.hashnode.dev/throwable-class-and-hierarchy